### ISSUE TITLE
[Meta] Add labels for each component

### ISSUE THREAD

**Issue by kylemh** [radix-ui/primitives#1204]
## Documentation

I'd love to be able to quickly filter open and closed issues for a specific component. It feels like a good idea to add labels for each component.

**Comment by benoitgrelard** [comment]
We usually do `[DropdownMenu]` in the issue title so you can use that to filter.

I agree it might be nice to create proper label, but I also like the fact that it's part of the title, otherwise in certain views where the labels aren't visible you have no context as to what the issue is about.

So I'm not sure what to do.

**Comment by kylemh** [comment]
both sounds like a great idea, but only if you want to do this!

**Comment by benoitgrelard** [comment]
I guess my worry is that it kinda feels like duplicated effort.

**Comment by kylemh** [comment]
True. Which views exist where you don't see labels? Notifications?

One advantage labels have over titles is that you can have multiple labels. A PR or issue may concern many components or concepts at once.

You're the maintainer though. If you're happy with what you got, then that's all there is to it!
